fe2-android/app/src/main/java/com/github/dedis/popstellar/model/objects/Chirp.java

```diff
@@ -0,0 +1,98 @@
+package com.github.dedis.popstellar.model.objects;
+
+import androidx.annotation.NonNull;
+
+/** Class modeling a Chirp */
+public class Chirp {
+
+  private static final int MAX_CHIRP_CHARS = 300;
+
+  private String id;
+  private String channel;
+
+  private String sender;
+  private String text;
+  private long timestamp;
+  private int likes;
+  private String parentId;
+
+  public Chirp(String id) {
+    if (id == null) {
+      throw new IllegalArgumentException(" The id is null");
+    } else if (id.isEmpty()) {
+      throw new IllegalArgumentException(" The id of the Chirp is empty");
```

Comment:
There is an extra space in the string
```suggestion
      throw new IllegalArgumentException("The id is null");
    } else if (id.isEmpty()) {
      throw new IllegalArgumentException("The id of the Chirp is empty");
```